apix-rest-v1/src/main/java/eu/xenit/apix/rest/v1/internal/ApixArgumentResolver.java

```diff
@@ -33,11 +34,21 @@ public final Object resolveArgument(final Class<?> argumentType, final Annotatio
                     argumentType, expectedParameterType));
         }*/
 
-        //TODO: inject apix object mapper?
-        ObjectMapper map = new ObjectMapper();
-
         try {
-            return map.readValue(request.getContent().getContent(), argumentType);
+            if (Exception.class.isAssignableFrom(argumentType)) {
```

Comment:
Can't you just not use this `ArgumentResolver` for `Exception`s? 

DE wil automatically resolve an exception parameter for `@ExceptionHandler` annotated methods.

Reply:
Just to be clear: this comment is not related to moving from `ApixArgumentResolver` to `@RequestBody`. Exception arguments for methods annotated with `@ExceptionHandler(...)` do not require the `@RequestBody` annotation.

Reply:
I indeed end up in that resolver through DE, but it is not entirely clear to me how DE has selected that resolver. I suspect that the issue might be that it reuses the resolver it selected to handle the original incoming request.
It is also not clear to me how I'd configure DE to select another resolver (If my earlier hypothesis is correct, I dont think we can switch to another resolver for the existience of that request)

Reply:
You can indicate if DE should use the resolver via the `supports(final Class<?> parameterType, final Class<? extends Annotation> annotationType)` method. 

AS-IS this resolver is applicable for the `FileExistsException` because the package of that class `.startsWith("eu.xenit.apix")`

My suggestion is to adapt the  `supports(...)` method so the `ApixArgumentResolver` is only used for model classes or at least not for Exceptions.

Reply:
aah, I see.
I misunderstood and changed the wrong method. ill give it a try with changing `supports`